### ISSUE TITLE
Dashed attributes not converted to camelCase

### ISSUE THREAD

**Issue by aaron-harvey** [aknuds1/html-to-react#6]
It seems like HTML / SVG elements containing dashed attributes (e.g. `stroke-width`) are not being converted to camelCase causing React to complain and throw out the props.

```javascript
it('should handle dashed attributes', function () {
    var input = '<svg><circle stroke-width="2"></circle></svg>';
    var reactComponent = parser.parse(input);
    var reactHtml = ReactDOMServer.renderToStaticMarkup(reactComponent);

    assert.equal(reactHtml, input);
});
```
```diff
AssertionError: '<svg><circle></circle></svg>' == '<svg><circle stroke-width="2"></circle></svg>'
+ expected - actual
-<svg><circle></circle></svg>
+<svg><circle stroke-width="2"></circle></svg>
```

**Comment by aknuds1** [comment]
Would you mind adding the test via a pull request? Thanks!

**Comment by aaron-harvey** [comment]
Addressed with #7
